src/i18n/es-es/index.js

```diff
@@ -343,6 +343,7 @@ export default {
             address_copied: 'Dirección copiada al portapapeles',
             search_placeholder: 'Dirección, Tx, Bloque',
             select_language: 'Seleccionar idioma',
+            open_language_switcher: '',
```

Comment:
Now reading the Portuguese version I understand the "Open" word. 
You should change this as follows
```suggestion
            open_language_switcher: 'Abrir el selector de idioma',
```